### ISSUE TITLE
TVN24

### ISSUE THREAD

**Issue by Huero23** [mbebe/blomqvist#478]
Fakty po faktach nie działają :( teraz tylko płatne ?

**Comment by mbebe** [comment]
Nie. Byly zmiany. Poszla poprawka. Wersja tez juz dziala pod 19.

**Comment by Huero23** [comment]
Czy poprawka działa tylko pod Kodi 19 ? Weszłem na Fakt Po Faktach jest pusto ?

**Comment by mbebe** [comment]
Sorry, mialem maly blad.
Poszla poprawka.

**Comment by Huero23** [comment]
@mbebe Dziękuje teraz pokazuje się lista tylko jak clickam na przykład na "piątek 26 lutego" żeby zobaczyć cały program fakty po faktach to nic nie rusza dolne link działają dobrze ale gdzie wejść żeby zobaczyć cały program ?

**Comment by mbebe** [comment]
Poszla kolejna poprawka.

**Comment by Huero23** [comment]
@mbebe TVN Fakty 👍 Wszytko działa! 🎯💯 Dziękuje bardzo !
